docs/source/evaluation.md

```diff
@@ -37,9 +37,10 @@ For more info see: {ref}`is_standard`
 
 You should score your TRE against each statement in the SATRE specification using this scoring system:
 
-:0 Not met: The TRE does not meet this requirement (the TRE is not SATRE compliant)
+:0 Not met: The TRE does not meet this requirement (if this is **Mandatory** this means the TRE is not SATRE compliant)
 :1 Sufficient: The TRE meets this requirement met but there is substantial scope for improvement
 :2 Satisfied: The TRE meets this requirement met but there may still be scope for improvement
+:N/A: Not applicable: The statement is not relevant to a TRE, may apply to **Recommended** or **Optional** statements
```

Comment:
Note if we return to https://github.com/sa-tre/satre-specification/pull/272 and use conditional Mandatory statements this will need to be updated, AFAIK currently all Mandatory statements are mandatory for all.

Reply:
I think this is right, there's probably statements in out eval where we gave ourselves a 2 and just said "we pass because it's not applicable" - I don't think this is a huge issue though